stellarphot/io/tess.py

```diff
@@ -213,33 +213,24 @@ def invalid_parts(self):
 class TOI:
     """ A class to hold information about a TOI (TESS Object of Interest).
 
-    Parameters
-    ----------
-
-    tic_id: int
-        The TIC ID of the target.
-
-    toi_table: str, optional
-        The path to the TOI table. If not provided, the default table will be downloaded.
-
-    allow_download: bool, optional
-        Whether to allow the default table to be downloaded if it is not found.
-
     Attributes
     ----------
 
-    tess_mag: float
-        The TESS magnitude of the target.
-
-    tess_mag_error: float
-        The uncertainty in the TESS magnitude.
+    coord: `~astropy.coordinates.SkyCoord`
+        The coordinates of the target.
 
     depth: float
         The transit depth of the target.
 
     depth_error: float
         The uncertainty in the transit depth.
 
+    duration: float
```

Comment:
See preview comment about distinction between parameters to initialize and actual attributes (while redundant in this case, that is not always the case).

Reply:
Looking at https://numpydoc.readthedocs.io/en/latest/format.html#documenting-classes for the 3rd or 4th time today, I think you are right. In many cases the parameters will also be attributes, but not always (and not all attributes are parameters for sure).

It looks like the only case where you can save space is when the attribute is a `@property` -- then just listing the attribute name is enough because it grabs the docstring of the `@property`.

Reply:
This means I should probably go through and look for @property decorators and adjust do,strings appropriately.